automation/unit-tests.sh

```diff
@@ -25,6 +25,8 @@ for template in $templates; do
   oc process -f "$template" NAME=test DATA_SOURCE_NAME=test || exit 1
 done
 
+echo "generate all templates after cleaning up first"
+oc delete -f dist/templates || true
```

Comment:
This would not delete templates which differ from the ones in `dist/templates` (see L51). What about deleting the namespace that was created above after a test run?

Reply:
Done.